lib/protoflow-blocks/src/blocks/hash.rs

```diff
@@ -17,6 +17,9 @@ pub mod hash {
 
     pub trait HashBlocks {
         fn hash_blake3(&mut self) -> Hash;
+        fn hash_sha256(&mut self) -> Hash;
+        fn hash_sha1(&mut self) -> Hash;
+        fn hash_md5(&mut self) -> Hash;
```

Comment:
Its a better design to have each of these algos using a trait which it implements, not part of something greater. They are part of a family, but not inherently part of the same system.

Reply:
done